Framework/Algorithms/src/Stitch1DMany.cpp

```diff
@@ -20,26 +21,32 @@ DECLARE_ALGORITHM(Stitch1DMany)
 void Stitch1DMany::init() {
 
   declareProperty(
-      make_unique<ArrayProperty<std::string>>("InputWorkspaces",
-                                              Direction::Input),
-      "Input Workspaces. List of histogram workspaces to stitch together.");
+      Kernel::make_unique<ArrayProperty<std::string>>(
+          "InputWorkspaces",
+          boost::make_shared<MandatoryValidator<std::vector<std::string>>>()),
+      "Input Workspaces. List of histogram workspaces to stitch together. At "
+      "least 2 workspaces must be supplied for stitching and all must be of "
+      "the same type.");
 
   declareProperty(make_unique<WorkspaceProperty<Workspace>>(
                       "OutputWorkspace", "", Direction::Output),
                   "Output stitched workspace.");
 
   declareProperty(make_unique<ArrayProperty<double>>(
-                      "Params", boost::make_shared<RebinParamsValidator>(true),
+                      "Params", boost::make_shared<RebinParamsValidator>(false),
```

Comment:
The `Params` property was effectively mandatory in the original code. I believed it would be better to change the `Params` validator to make entering it mandatory rather than let the `validateCommonInputs()` method do it, as it makes it more explicit in the documentation that this property is mandatory.